src/Internal/Button/Implementation.elm

```diff
@@ -170,6 +178,9 @@ button domId lift model options nodes =
 
               else
                 []
+            , config.label
+                |> Maybe.map (\label_ -> [ Html.span [ Html.class "mdc-button__label" ] [ Html.text label_ ] ])
+                |> Maybe.withDefault []
             , nodes
```

Comment:
Hi @dpinn, would there be any drawback in wrapping `nodes` within `span.mdc-button__label`?

Reply:
Hey yeah! That would be even neater perhaps. I'll try out and get back to you.